Doc/library/tempfile.rst

```diff
@@ -211,6 +214,9 @@ The module defines the following user-callable items:
    .. versionchanged:: 3.12
       Added the *delete* parameter.
 
+   .. versionchanged:: 3.13
```

Comment:
Shouldn't new path property be explained somewhere?

Reply:
I thought same, but even the `.name` is not explained in this file. I can add it if someone can point an apt location 👍

Reply:
`NamedTemporaryFile` indeed had a reference to `.name`, I added `.path` there. Thanks!, ref: [diff](https://github.com/python/cpython/compare/5d6a344082136c396ee3a47068cd047b48c8a59c..840f7699060e6e3e11273c7c574784c92a92b2ef)